octoprint_psucontrol/__init__.py

```diff
@@ -204,6 +198,7 @@ def _configure_gpio(self):
             try:

```

Comment:
This whole "if self.enableOnOffButton" block should be after configuring the switching method as there could be a race condition when using GPIO switching.